src/Stepper/StepperHeader.jsx

```diff
@@ -71,11 +72,15 @@ StepperHeader.propTypes = {
   className: PropTypes.string,
   /** A component that receives `activeStepIndex` and `totalSteps` props to display them. */
   PageCountComponent: PropTypes.elementType,
+  /** Specifies whether the `Stepper` headers is clickable, if `true` appropriate `hover` and `focus` styling
+   * will be added and opportunities toggle content. */
```

Comment:
nit: what is meant by "opportunities toggle content"?

Reply:
This is redundant and incomprehensible, deleted